### PR TITLE
Change "Sending..."+spinner to "Sending... x/y" number of current number/total number of messages

### PR DESCRIPTION
**Pull request by amyzhao1321**
Change "Sending..."+spinner to "Sending... x/y" number of current number/total number of messages

### CODE REVIEW

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx [hunk 1]
```diff
@@ -172,7 +187,7 @@ class Messages extends React.Component<IMessageProps, IMessageState> {
           </Text>
         </Flex.Item>
         <Flex.Item size="size.quarter" variables={{ 'size.quarter': '24%' }}>
-          {this.sendIndicator(message)}
+          {this.sendingNumbers(message)}
```
Comment (jotrick) [inline Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx]: Does this.sendIndicator() need to be deleted?

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx [hunk 2]
```diff
@@ -161,6 +161,21 @@ class Messages extends React.Component<IMessageProps, IMessageState> {
     }
   }
 
+  private sendingNumbers = (message: any) => {
+    let currentNum = message.succeeded + message.failed + message.throttled;
```
Comment (jotrick) [inline Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx]: Move this line into the "else" block just above the return.

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx [hunk 3]
```diff
@@ -161,6 +161,21 @@ class Messages extends React.Component<IMessageProps, IMessageState> {
...
+    if (message.isCompleted) {
+      return (<Text />);
+    } else {
+      return (
+        <Text
+          truncated
+          content={`Sending...  ${currentNum} / ${message.totalMessageCount}`}
```
Comment (jotrick) [inline Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx]: Design said they actually want this to look like "Sending... 50 of 1000"
Just changing the '/' to the word "of".

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx [hunk 4]
```diff
@@ -161,6 +161,21 @@ class Messages extends React.Component<IMessageProps, IMessageState> {
     }
   }
 
+  private sendingNumbers = (message: any) => {
```
Comment (jotrick) [inline Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx]: Let's change this name to renderSendingNumbersText()

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/actions/index.ts [hunk 5]
```diff
@@ -9,6 +9,7 @@ type Notification = {
     succeeded: number,
     throttled: number,
     title: string,
+    totalMessageCount: string,
```
[the remark this answers is not in the document]
Comment (amyzhao1321) [reply Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/actions/index.ts]: done

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx (hunk 1 above)
Comment (amyzhao1321) [reply Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx]: deleted

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx (hunk 2 above)
Comment (amyzhao1321) [reply Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx]: done

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx (hunk 3 above)
Comment (amyzhao1321) [reply Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx]: done

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx (hunk 4 above)
Comment (amyzhao1321) [reply Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx]: done

**Pull request merged**